Training/Easy/defibrillators.rb

```diff
@@ -39,18 +39,18 @@ def find_closest(defibrilators_hash)
     closest_name
   end
 
-  def init_defibrilators(defibrilator)
-    d_long = parse(defibrilator[4])
-    d_lat = parse(defibrilator[5])
+  def init_defibrillators(defibrillator)
```

Comment:
If the function is called `init_defibrillators`, it should take all the objects at once. If you're initializing one, it's better to make the method singular.

Also, `add_defibrillator` might be a better name, because you're adding it to the existing `defibrillators` hash